ocean_lib/web3_internal/contract_base.py

```diff
@@ -12,6 +12,7 @@
 from eth_typing import BlockIdentifier
 from hexbytes import HexBytes
 from ocean_lib.enforce_typing_shim import enforce_types_shim
+from ocean_lib.exceptions import VerifyTxFailed
```

Comment:
here as well, there's an unused import.

Reply:
@mariacarmina this is still unused, I think. Maybe you missed this comment :)

Reply:
A ok, sorry, missed that one.